docs/development/developer-guide.rst

```diff
@@ -131,3 +131,35 @@ simple Python script ``tools/vendor.py``.
 To refresh the dependencies, run the following command::
 
     $ tox -e vendor
+
+----------------
+Type annotations
+----------------
+
+Most standards and best practices are enforced by
+[Ruff](https://docs.astral.sh/ruff/rules/)'s ``ANN2``, ``FA``, ``PYI``, ``UP``
```

Comment:
`ANN2` added in https://github.com/pypa/setuptools/pull/4409/

Reply:
I did not find the ``FA`` rules, is that PyFlake (`F`)?